### ISSUE TITLE
why validation_samples = 848824?

### ISSUE THREAD

**Issue by deeplearningzhy** [zhr1201/CNN-for-single-channel-speech-enhancement#13]
Thanks for your sharing,but I am confused that why validation_samples = 848824?
I download the TIMIT data set, for train set are 4680 waves,for test set are 1620 waves,for Nonspeech are 100 types of noise,could you please help me to figure out why validation samples are 848824? Thank you very much!

**Comment by ffxz** [comment]
@deeplearningzhy do you deal with the problem? I also confused about that.

**Comment by zhr1201** [comment]
Sorry about the delay, I just used the number of validation samples in my validation set, of course you can set the number to a new value once you set up your own validation set

**Comment by ffxz** [comment]
@zhr1201 the number of validation_samples is random? I can set it without calculating?
